engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -1209,12 +1194,21 @@ struct divine_toll_t : public paladin_spell_t
 
 struct hallowed_discernment_tick_t : public paladin_spell_t
 {
-  double aoe_multiplier;
   hallowed_discernment_tick_t( paladin_t* p ) : paladin_spell_t( "hallowed_discernment", p, p->find_spell( 340203 ) )
   {
     base_multiplier *= p->conduit.hallowed_discernment.percent();
     background = true;
   }
+
+  double action_multiplier() const override 
+  {
+    double am = paladin_spell_t::action_multiplier();
+    // for some god forsaken reason this ability double dips with vers :)
+    // assumption is because it takes a tick of another ability that is already scaled with vers
+    // then runs it through damage code again which applies vers again
+    am *= (1 + p()->composite_damage_versatility());
```

Comment:
Since this is (most likely) bugged behaviour, you could gate it behind a p() -> bugs check.

(Don't worry it's a bool that is enabled by default)

Reply:
✔️